### DISCUSSION TITLE
[Lab1,2] 報告要包含甚麼?

### DISCUSSION THREAD

**Discussion by jiyaunran** [bol-edu/HLS-SOC-Discussions#10] [Q&A] [answered]
我看到submission裡面要有report。請問上面說report要包含
‐ Performance
‐ Utilization
‐ Interface
這些是要從哪裡看呢?

**Comment by tofumanjeff** [comment, marked as answer]
You can find them in synthesis reports (_csynth.rpt_ and {your_function_name}__csynth.rpt_).

**Comment by kevin861222** [comment]
You can use Vivado to find the post-synthesis and the post-implementation utilization table | graph in the project summary.
![截圖 2023-09-17 上午10 48 04](https://github.com/bol-edu/HLS-SOC-Discussions/assets/79128379/93fdc7fa-6fb8-4174-929b-f6cbb202e101)
